### PR TITLE
 Kd alignment for Offsite-tuning

### PR DESCRIPTION
**Pull request by rayrayraykk**
Alignment only after generate Emulator (align during FedAvg is impossible due to high communication cost).

### CODE REVIEW

File: federatedscope/llm/baseline/llama_offsite_align.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  local_update_steps: 10
+  batch_or_epoch: batch
+  optimizer:
+    lr: 0.0001
+    weight_decay: 0.0
+#  is_enable_half: True
+criterion:
+  type: CrossEntropyLoss
+trainer:
+  type: llmtrainer
```
Comment (HarliWu) [inline federatedscope/llm/baseline/llama_offsite_align.yaml:50]: Should it be KDTrainer?

File: federatedscope/llm/offsite_tuning/server.py [hunk 2]
```diff
@@ -39,6 +40,16 @@ def __init__(self,
...
+        # Emulator alignment
+        if config.llm.offsite_tuning.emu_align.use:
+            adap_model = align_student_with_teacher(raw_model=model,
+                                                    adap_model=adap_model,
+                                                    cfg=config,
+                                                    device=device,
+                                                    monitor=Monitor(
+                                                        config,
+                                                        monitored_object=self))
+
```
Comment (HarliWu) [inline federatedscope/llm/offsite_tuning/server.py:52]: Please consider adding the following code: 

```
else:
    adap_model = generate_emulator_and_adapter(model, strategy=compress_stratgegy, emulator_l=emulator_l, emulator_r=emulator_r, **offsite_tuning_kwargs)
```

**Review by HarliWu — APPROVED** [review]
LGTM

**Review by qbc2016 — APPROVED** [review]
LGTM.

**Pull request merged**